packages/dashboard/testing/mocks.ts

```diff
@@ -176,6 +177,52 @@ export const MOCK_RECTANGLE_WIDGET: RectangleWidget = {
   },
 };
 
+export const MOCK_LINE_WIDGET: LineWidget = {
+  id: 'mock-line-widget',
+  type: 'line',
+  x: 0,
+  y: 0,
+  z: 1,
+  width: 7,
+  height: 4,
+  properties: {
+    start: {
+      x: 25,
+      y: 200,
+    },
+    end: {
+      x: 375,
+      y: 200,
+    },
+    lineStyle: 'solid',
+    color: 'black',
+    thickness: 5,
+  },
+};
+
+export const MOCK_LINE_WIDGET_2: LineWidget = {
```

Comment:
do we need both of these mocks? they are the same except for the id?

Reply:
i may have added that to test something and ended up forgetting. removed it.